pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[project]
```

Comment:
Under project, please add:
requires-python = ">= 3.9"

Reply:
It's already there.